src/pages/signup/SignupPage.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+// import { Stepper, Step, StepLabel, Typography, Button } from "@mui/material-ui/core";
+import Box from '@mui/material/Box';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import TextField from '@mui/material/TextField';
+
+import StepOne from './StepOne';
+import StepTwo from './StepTwo';
+import StepThree from './StepThree';
+import StepFour from './StepFour';
+
+const steps = [
+  "Your personal information",
+  "Create your password",
+  "Your Loan information",
+  "Your bank verification"
+];
+
+const SignupPage = () => {
+
+  function showSteps(step) {
+    switch(step) {
+      case 1 :
+        return <StepOne />
+      case 2 :
+        return <StepTwo />
+      case 3 :
+        return <StepThree />
+      case 4 :
+        return <StepFour />
+    }
+  }
+
+  return (
+    <div className="form-signup">
+      <Stepper 
+        activeStep={1} 
```

Comment:
The active step here is hard-coded, but we want to progress through the different steps. We could plug in some state for the `activeStep` using our useState hook?

Reply:
When the user clicks the next button, the activeStep count needs to increment